LKMEX-whitelist/Bullish_Rockers_Lkmex_Whitelist.md

```diff
@@ -0,0 +1,35 @@
+## Name of the project
+Bullish Rockers Club
+
+## Description of the project
+Bullish Rockers is a music themed NFT Collection on MultiversX composed of 10,000 Unique Bulls with a purpose to connect NFT community to new Rock Bands
+
+website: https://bullishrockersclub.com/
+
+## Purpose
+We send to our holders biweekly LKMEX rewards
+
+To farm LKMEX on the dex we use the following wallet erd1yjvtlgrhdw7svfzryaa6srkgmfls3llp3w465nsw0qnnh69w5hrscvn4ua
```

Comment:
We do not whitelist user addresses. Please collaborate with other projects and use a smart contract.